recipes/timml/meta.yaml

```diff
@@ -0,0 +1,43 @@
+# Jinja variables help maintain the recipe as you'll update the version only here
+{% set name = "timml" %}
+{% set version = "5.0.0" %}
+
+package:
+   name: {{ name|lower }}
+   version: {{ version }}
+
+source:
+  git_rev: v{{ version }}
+  git_url: https://github.com/mbakker7/timml.git
+
+build:
+  number: 0
+  skip: True  # [py<34]
+  string: py36_0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
It doesn't look like the `setup.py` uses `setuptools`, so `--single-version-externally-managed` isn't going to be available.